cuegui/cuegui/plugins/LogViewPlugin.py

```diff
@@ -862,3 +867,78 @@ def __init__(self, parent=None):
             self, parent, PLUGIN_NAME, QtCore.Qt.RightDockWidgetArea)
         self.logview_widget = LogViewWidget(self)
         self.layout().addWidget(self.logview_widget)
+
+
+class Highlighter(QtGui.QSyntaxHighlighter):
+    def __init__(self, parent=None):
+        super(Highlighter, self).__init__(parent)
+
+        self.on = True
+
+        self.timeFormat = QtGui.QTextCharFormat()
+        self.timeFormat.setFontWeight(QtGui.QFont.Bold)
+        self.timeFormat.setForeground(cuegui.Style.ColorTheme.LOG_TIME)
+
+        self.errorFormat = QtGui.QTextCharFormat()
+        self.errorFormat.setFontWeight(QtGui.QFont.Bold)
+        self.errorFormat.setForeground(cuegui.Style.ColorTheme.LOG_ERROR)
+
+        self.warnFormat = QtGui.QTextCharFormat()
+        self.warnFormat.setFontWeight(QtGui.QFont.Bold)
+        self.warnFormat.setForeground(cuegui.Style.ColorTheme.LOG_WARNING)
+
+        self.infoFormat = QtGui.QTextCharFormat()
+        self.infoFormat.setFontWeight(QtGui.QFont.Bold)
+        self.infoFormat.setForeground(cuegui.Style.ColorTheme.LOG_INFO)
+
+        self.completeFormat = QtGui.QTextCharFormat()
+        self.completeFormat.setFontWeight(QtGui.QFont.Bold)
+        self.completeFormat.setForeground(cuegui.Style.ColorTheme.LOG_COMPLETE)
+
+
+    def highlightBlock(self, text):
+        if not self.on:
+            return
+
+        line = text.lower()
+        done = False
+
+        for error in ['error','aborted','fatal', 'failed', 'killed', 'command not found']:
```

Comment:
I'm thinking we should move this list, and the lists of things which are detected as warnings and progress lines, to `Constants.py` instead of hardcoding them here. That should make it easier to customize.

Eventually we will need to figure out a better way to allow customization of these things, via config files or something like that, but I think having them as constants is good enough for now.